capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -0,0 +1,23 @@
+package com.google.univiz.scorecard;
+
+/**
+ * CollegeData is an interface that contains methods and functions which communicate with the
+ * CollegeScorecardAPI to obtain college data from the URL in JSON format.
+ */
+public interface CollegeData {
+
+  /** TODO(biancamacias): decide what getSuggestions does */
+  public List<ScorecardData> getSuggestions(String suggestion) {}
+
+  /**
+   * getCollegesById takes ids and field names of colleges selected and return a list of
+   * ScorecardData to be used by the servlets.
+   */
+  public List<ScorecardData> getCollegesById(List<String> ids, List<String> fieldNames) {}
+
+  /**
+   * scorecardApiCall takes ids and field names of colleges selected to call the CollegeScorecardAPI
+   * and return the data as JSON.
+   */
+  public String scorecardApiCall(List<String> ids, List<String> fieldNames) {}
```

Comment:
is this supposed to return a JSON object?

Reply:
yeah a string that holds the JSON response? but i havent done the implementation, i might have to use the ```JSONobject``` type.